rollup/Makefile

```diff
@@ -4,8 +4,8 @@ IMAGE_VERSION=latest
 REPO_ROOT_DIR=./..
 
 mock_abi:
-	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --sol ./rollup/mock_bridge/MockBridgeL1.sol --pkg mock_bridge --out ./rollup/mock_bridge/MockBridgeL1.go
-	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --sol ./rollup/mock_bridge/MockBridgeL2.sol --pkg mock_bridge --out ./rollup/mock_bridge/MockBridgeL2.go
+	cd .. && solc --evm-version cancun --bin --abi --optimize --overwrite -o ./build/bin ./rollup/mock_bridge/MockBridge.sol
```

Comment:
I also noticed this. When solving this issue, I found that the unit tests do not use `mock L2 contract` anywhere (gas-oracle only checks db update), thus removing `mock L2 contract` temporarily (otherwise with the new `abigen` command, some Golang function names will conflict, need to restructure the mock_bridge dir's tree structure).

Will add it back when it's used.